src/data/moves_info.h

```diff
@@ -15192,14 +15205,16 @@ const struct MoveInfo gMovesInfo[MOVES_COUNT_ALL] =
         .description = COMPOUND_STRING(
             "Throws 2 to 5 stars that\n"
             "are sure to strike first."),
-        .effect = EFFECT_MULTI_HIT,
+        .effect = EFFECT_CHANGE_POWER_ON_SPECIES,
         .power = 15,
         .type = TYPE_WATER,
         .accuracy = 100,
         .pp = 20,
         .target = MOVE_TARGET_SELECTED,
         .priority = 1,
         .category = B_UPDATED_MOVE_DATA >= GEN_7 ? DAMAGE_CATEGORY_SPECIAL : DAMAGE_CATEGORY_PHYSICAL,
+        .argument = { .powerBySpecies.species = SPECIES_GRENINJA_BATTLE_BOND, .powerBySpecies.power = 20, },
```

Comment:
```suggestion
        .argument = { .powerBySpecies.species = SPECIES_GRENINJA_ASH, .powerBySpecies.power = 20, },
```
`SPECIES_GRENINJA_BATTLE_BOND` is the regular Greninja whose only ability is Battle Bond.
Also, thoughts on having the 3 hit effect also as a part of this move effect?